### ISSUE TITLE
How to give product feedback 

### ISSUE THREAD

**Issue by tomresing** [MicrosoftDocs/visualstudio-docs#594]
Is there a better link for the new product feedback button? With the [new feedback system pilot](https://docs.microsoft.com/en-us/teamblog/a-new-feedback-system-is-coming-to-docs) on the [Visual Studio IDE article](https://docs.microsoft.com/visualstudio/ide/visual-studio-ide), the product feedback link leads directly to [developercommunity.visualstudio.com](https://developercommunity.visualstudio.com). 

The developer community does seem like the right community for product feedback. However, the home page has very little to no guidance on how to provide product feedback. [How to report a problem with Visual Studio 2017](https://docs.microsoft.com/en-us/visualstudio/ide/how-to-report-a-problem-with-visual-studio-2017) provides specific instructions, but is focused on problems rather than generic feedback.

**Comment by TerryGLee** [comment]
@tomresing Thanks for your comments about the new Feedback tool. We do have content that addresses other ways to give feedback; specifically, we have a [Talk to us](https://docs.microsoft.com/visualstudio/ide/talk-to-us) page, where the information you seek is in the "I want to make a suggestion about Visual Studio features" section. (The short of that is that the [UserVoice](https://visualstudio.uservoice.com/forums/121579-visual-studio-ide) site is the correct place to do this.) 

That being said, we'll look at how we can surface that information more intuitively in our documentation.

Thanks again for your comments!

**Comment by tomresing** [comment]
Thanks. I submitted the pull request above based on your feedback and the guidance on product feedback issues: https://review.docs.microsoft.com/en-us/new-hope/resources/github-issues-feedback

**Comment by TerryGLee** [comment]
@tomresing We'll review the PR; thanks. (And, I see that you are a Microsoft employee, so I'll respond further to you via internal email.)

**Comment by TerryGLee** [comment]
@tomresing Although we have opted not to accept your PR, we will address adding more information to the [How to Report a Problem](https://docs.microsoft.com/en-us/visualstudio/ide/how-to-report-a-problem-with-visual-studio-2017) page so that it includes more info about where to send us generic product feedback or a suggestion for a product improvement.

**Comment by TerryGLee** [comment]
@tomresing We've updated the [How to report a problem in Visual Studio 2017](https://docs.microsoft.com/en-us/visualstudio/ide/how-to-report-a-problem-with-visual-studio-2017) page; specifically, we've revised the "Alternate reporting" section to include a links and guidance on when to use [UserVoice](https://visualstudio.uservoice.com/forums/121579-visual-studio-ide) to contribute product feedback and suggestions vs. when to use the [Developer Community](https://developercommunity.visualstudio.com/) to report a problem.  

As to how our GitHub Issues feedback mechanisms work, we'll handle that separately (and per our email exchange). 

Thanks,
~ Terry
